Remotion/Web/Development.WebTesting/Configuration/ITestInfrastructureConfiguration.cs

```diff
@@ -36,17 +36,6 @@ public interface ITestInfrastructureConfiguration
     /// </summary>
     string ScreenshotDirectory { [CanBeNull] get; }
 
-    /// <summary>
-    /// Specifies how long the Coypu engine should maximally search for a web element or try to interact with a web element before it fails.
-    /// </summary>
-    TimeSpan SearchTimeout { get; }

```

Comment:
Document removed properties as two separate breaking changes.
When backporting, do not remove the properties from the interface. Instead, let them return the correct values from DriverConfiguration to avoid breaking existing code bases during a hotfix upgrade.